staging/src/k8s.io/kube-aggregator/pkg/cmd/server/start.go

```diff
@@ -49,27 +49,26 @@ type AggregatorOptions struct {
 }
 
 // NewCommandStartAggregator provides a CLI handler for 'start master' command
-func NewCommandStartAggregator(out, err io.Writer, stopCh <-chan struct{}) *cobra.Command {
-	o := NewDefaultOptions(out, err)
-
+// with a default AggregatorOptions.
+func NewCommandStartAggregator(defaults *AggregatorOptions, stopCh <-chan struct{}) *cobra.Command {
```

Comment:
`o := *defaults`. Now, operating on defaults is strange.

Reply:
ic